### PR TITLE
[v2.0.0] staging -> main

### PR DESCRIPTION
**Pull request by swarna04**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -17,6 +17,9 @@ clean:
 	(rm -rf ci)
 	(rm -rf $(AAR_FILE_DIR))
 	(./code/gradlew -p code clean)
+	
+checkstyle:
+	(./code/gradlew -p code/optimize checkstyle)
```
Comment (swarna04) [inline Makefile:22]: Just noticed this here, we should fix this with next release 
```
(./code/gradlew -p code/$(EXTENSION-LIBRARY-FOLDER-NAME checkstyle)
```

File: code/optimize/src/test/java/com/adobe/marketing/mobile/optimize/OptimizeTests.java [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
  Unless required by applicable law or agreed to in writing, software distributed under
  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
- OF ANY KIND, either express or implied. See the License for the specific language
+ OF ArgumentMatchers.any KIND, either express or implied. See the License for the specific language
```
Comment (swarna04) [inline code/optimize/src/test/java/com/adobe/marketing/mobile/optimize/OptimizeTests.java:9]: :(

File: code/optimize/src/test/java/com/adobe/marketing/mobile/optimize/OptimizeExtensionTests.java [hunk 3]
```diff
@@ -6,631 +6,687 @@
 
  Unless required by applicable law or agreed to in writing, software distributed under
  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
- OF ANY KIND, either express or implied. See the License for the specific language
+ OF ArgumentMatchers.any KIND, either express or implied. See the License for the specific language
```
Comment (swarna04) [inline code/optimize/src/test/java/com/adobe/marketing/mobile/optimize/OptimizeExtensionTests.java:9]: Here too :(

File: code/optimize/src/test/java/com/adobe/marketing/mobile/optimize/OfferTests.java [hunk 4]
```diff
@@ -6,47 +6,34 @@
 
  Unless required by applicable law or agreed to in writing, software distributed under
  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
- OF ANY KIND, either express or implied. See the License for the specific language
+ OF ArgumentMatchers.any KIND, either express or implied. See the License for the specific language
```
Comment (swarna04) [inline code/optimize/src/test/java/com/adobe/marketing/mobile/optimize/OfferTests.java:9]: A third time :(

File: code/optimize/src/test/java/com/adobe/marketing/mobile/optimize/DecisionScopeTests.java [hunk 5]
```diff
@@ -6,301 +6,347 @@
 
  Unless required by applicable law or agreed to in writing, software distributed under
  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
- OF ANY KIND, either express or implied. See the License for the specific language
+ OF ArgumentMatchers.any KIND, either express or implied. See the License for the specific language
```
Comment (swarna04) [inline code/optimize/src/test/java/com/adobe/marketing/mobile/optimize/DecisionScopeTests.java:9]: ⬆️

**Review by spoorthipujariadobe — APPROVED** [review]

**Pull request merged**